### ISSUE TITLE
does this plugin work with matrtix?

### ISSUE THREAD

**Issue by jasondaigo** [jellyfin/jellyfin-plugin-webhook#122]
is there a template possibility for matrix-appservice-webhook? im running this: https://github.com/turt2live/matrix-appservice-webhooks   
does it work with such a webhook?

**Comment by crobibero** [comment]
Yes, this should work without issue. If you have something you are trying to accomplish please be more specific so I can assist in creating a template.

**Comment by hardwareadictos** [comment]
Would be great also to have a template for Matrix as we have for ntfy
